### PR TITLE
[SOL-1721] Change Total Paid Coupon Field

### PR DESCRIPTION
**Pull request by marjev**

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @marjev! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?number=648&repo=edx%2Fecommerce

**Comment by marjev** [comment]
@mjfrey Here's a PR for [SOL-1721](https://openedx.atlassian.net/browse/SOL-1721). Please review.

Here's the screenshot:
new-create-new-coupon-page

File: ecommerce/static/js/views/coupon_form_view.js [hunk 1]
```diff
@@ -247,23 +251,30 @@ define([
...
                 }
             },
 
+            changePrice: function () {
+                this.updatePrice(this.getSeatData());
+            },
+
+            updatePrice: function (seatData) {
+                var quantity = this.$el.find('input[name=quantity]').val(),
+                    price = quantity * seatData.price;
```
Comment (mjfrey) [inline ecommerce/static/js/views/coupon_form_view.js:273]: Shouldn't we be calculating the price less any discount?  If they are discount codes we need to take that into account.  The total price would be (unit price \* discount) \* quantity

Comment (marjev) [inline ecommerce/static/js/views/coupon_form_view.js:273]: Please discuss with @pbaruah. I just implemented the calculation that I found in the story description, which is:

> Calculation: Seat price X number of codes

**Comment by pbaruah** [comment]
@marjev , @mjfrey - this is the total value of the coupon before discount. This amount goes in the "Price" field in the downloadable CSV report. Could you please take out the word discount to read "number of coupon codes X unit price per seat"

**Comment by marjev** [comment]
@mjfrey Is it ok to merge this PR?

**Comment by mjfrey** [comment]
@marjev can you update the screen shot?  Did you change the text according to @pbaruah

**Comment by marjev** [comment]
@mjfrey yes, I did the update. Here's the new screenshot:

screen shot 2016-04-06 at 14 49 53

**Comment by mjfrey** [comment]
:+1:

**Pull request merged**